### PR TITLE
Removed unnecessay negative margin from email div

### LINKED ISSUE #3865: [Footer] Subscribe component overlaps on zoom greater than 100%
### Description
When a site visitor has their browser zoom level at a level higher than 100%, the subscribe component overlaps other elements in the footer.

### Expected Behavior
No overlap.

### Screenshots
Screenshot 2023-03-02 at 12 32 40 PM

### Environment:
- Host OS: macOS
- Browser: Safari

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by chiragchhabria2**
**Description**

This PR fixes #3865 

**Notes for Reviewers**
Removed -7 margin from email div

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit bf613042ac5bb4bf83a279eb68111f760ac1b460 at: https://64412e5cd618d02449153530--layer5.netlify.app

**Comment by chiragchhabria2** [comment]
@l5io Thank you!

**Comment by chiragchhabria2** [comment]
What should I do about these issues?

**Comment by chiragchhabria2** [comment]
> > @Shivam-AfA no I didn't. I checked the issue on chrome desktop and mobile view only.
> 
> @chiragchhabria2 This issue is only reproducible on Safari, as mentioned by @Nikhil-Ladha. Do you see the same issue on chrome as well?

Yes I did see same issue even in chrome if you zoom in 150% or more

**Comment by Shivam-AfA** [comment]
@chiragchhabria2 That's not the case for me. Maybe you can present this issue and the solution on today's website call?

**Comment by chiragchhabria2** [comment]
> @chiragchhabria2 That's not the case for me. Maybe you can present this issue and the solution on today's website call?

What's a website call? And how do I attend it?

File: src/sections/General/Footer/footer.style.js [hunk 1]
```diff
@@ -148,12 +148,6 @@ const FooterWrapper = styled.section`
 			display: none;
 		}
 	}
-
-	@media only screen and (max-width: 1199px) and (min-width: 704px) {
-		.subscribe {
-			margin-top: -7rem;
-		}
-	}
```
Comment (Nikhil-Ladha) [inline src/sections/General/Footer/footer.style.js:156]: This doesn't help becaus, if we remove the top margin then there is a big mpty space below the last 2 columns in the footetr for screen width < 1200px.
Secondly, I don't see the issue reproducible anyway on any of the browser as of now. Please share the reproducible video or so.

**Comment by Shivam-AfA** [comment]
> What's a website call? And how do I attend it?

@chiragchhabria2 The website's call takes place every Monday at 5:30 pm IST. Contributors present what they've been working on and ask for feedback. The link to the call is https://meet.layer5.io/websites. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) and let's get the final feedback from the community.

**Comment by chiragchhabria2** [comment]
@Shivam-AfA 
here are screenshots of issue , I have taken these directly from layer5.io in chrome : 
[screenshots](https://imgur.com/a/b55c3f6)
here are screenshots from [live](https://64412e5cd618d02449153530--layer5.netlify.app/) preview of my commit
[preview screenshots](https://imgur.com/a/6UUdCWD)

**Comment by chiragchhabria2** [comment]
Hey @Shivam-AfA did you see those screenshots?

**Comment by Shivam-AfA** [comment]
Hey @chiragchhabria2. Thanks for sharing the demos. Please present this issue and solution on Monday's website call, if you would. Join the slack community here: https://layer5io.slack.com

**Comment by chiragchhabria2** [comment]
I won't be able to I have my classes I can do it on Thursday's call

**Comment by Chadha93** [comment]
@chiragchhabria2 Seeing no activity here, let's close this issue. Feel free to reopen it, if you wish to continue.

**Pull request closed without merging**